src/styles/mixins/mixins.scss

```diff
@@ -0,0 +1,6 @@
+@use '../_variables/variables' as *;
+
+@mixin transition-options($properties...) {
+  transition-property: $properties;
+  transition-duration: $TransitionDuration;
```

Comment:
The variable `$TransitionDuration` is not defined in this file. Ensure that it is imported correctly from the variables file or defined within this file.